contrib/pmsnare/pmsnare.c

```diff
@@ -81,11 +84,10 @@ CODESTARTisCompatibleWithFeature
 		iRet = RS_RET_OK;
 ENDisCompatibleWithFeature
 
-
 BEGINparse
 	uchar *p2parse;
 	int lenMsg;
-	int snaremessage;
+	int len_snare_tag;
 	int tablength;
 
 CODESTARTparse
```

Comment:
These changes should effectively detect what escaping is set and set the tabRepresentation accordingly. They can be overridden, if desired, when configuring the module.